src/richie/apps/courses/migrations/0019_auto_20201113_1630.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.3 on 2020-11-13 15:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("courses", "0018_auto_20201102_1912"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="coursepluginmodel",
+            name="variant",
+            field=models.CharField(
+                blank=True,
+                choices=[(None, "Inherit"), ("large", "Large")],
+                help_text="Optional glimpse variant for a custom look.",
+                max_length=50,
+                null=True,
+                verbose_name="variant",
+            ),
+        ),
+    ]
```

Comment:
This migration is requested by Django (although useless) but you most importantly need a "data migration" to correct the field values See example here: https://github.com/openfun/richie/blob/master/src/richie/apps/courses/migrations/0013_migrate_data_translated_licence_fields.py